test/parallel/test-process-env-allowed-flags.js

```diff
@@ -62,15 +62,41 @@ const assert = require('assert');
                      true);
 
   process.allowedNodeEnvironmentFlags.add('foo');
+  Set.prototype.add.call(process.allowedNodeEnvironmentFlags, 'foo');
   assert.strictEqual(process.allowedNodeEnvironmentFlags.has('foo'), false);
```

Comment:
Non-blocking nit that you can totally ignore if you think it's unnecessary: It might be best to do the assertion after each call so that (a) you know whether `.add()` or `prototype.add.call()` broke the test, and (b) you avoid a wacky possibility where `add()` causes a bug that is masked by some unforeseen side effect of `prototype.add.call()`:

```suggestion
  process.allowedNodeEnvironmentFlags.add('foo');
  assert.strictEqual(process.allowedNodeEnvironmentFlags.has('foo'), false);
  Set.prototype.add.call(process.allowedNodeEnvironmentFlags, 'foo');
  assert.strictEqual(process.allowedNodeEnvironmentFlags.has('foo'), false);
```